src/test/java/seedu/address/logic/commands/PartnerCourseDeleteCommandTest.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands;
```

Comment:
After this review, maybe make rest of tests in separate PR?

Reply:
ok!